### ISSUE TITLE
Server crashing bug

### ISSUE THREAD

**Issue by RadianceD2** [ValveSoftware/Dota2-Gameplay#14425]
### Description

First of all how easy it is to do is a joke. Twice today wins didnt count while loses do (cause were not the type of people to abuse these types of things). Anyway hopefully this helps get this thing resolved.
Match ID from w33haa stream since i cannot find the game ID of my own games that got crashed. 
This is done withought the all chatting so most likely team chat spam of the random characters followed by the disconnect.
Please, please. MAXIMUM PRIORITY on this

### Example Match ID (and possibly Timestamp)

ID 7494875955 . time 27.40 

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/154266105/a77f10f7-08fc-4db8-bb3a-3eaec6d16c81)

**Comment by xstrei06** [comment]
yeah Valve should do something about this real fast, don't remember worse bug since ever, 5 wins today did not score

**Comment by Mahdi-zarei** [comment]
This thing is joke, people can crash the server by sending random long chats, and what valve DOES ban is switching accounts by steam, grats very clever

**Comment by snatch3154** [comment]
到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到到
